src/test/scala/dsptools/VerboseDspTesterSpec/SimpleTBwGenTypeOption.scala

```diff
@@ -349,7 +351,7 @@ class SimpleTBSpec extends FlatSpec with Matchers {
     val opt = new DspTesterOptionsManager {
       dspTesterOptions = optionsPass.dspTesterOptions
       testerOptions = optionsPass.testerOptions
-      commonOptions = optionsPass.commonOptions.copy(targetDirName = "test_run_dir/lit_sint")
+//      commonOptions = optionsPass.commonOptions.copy(targetDirName = "test_run_dir/lit_sint")
```

Comment:
I don't think so, I am working on a fix in chisel-testers, but it was taking to much time so I've jumped to this for now.  But I will get back to it.